internal/pkg/errorconcept/common.go

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright 2019 Dell Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *******************************************************************************/
+package errorconcept
+
+import (
+	"net/http"
+
+	"github.com/edgexfoundry/edgex-go/internal/core/metadata/errors"
+)
+
+var Common commonErrorConcept
+
+// CommonErrorConcept represents error concepts which apply across core-services
+type commonErrorConcept struct {
+	LimitExceeded       errLimitExceeded
+	DuplicateIdentifier duplicateIdentifier
```

Comment:
Shouldn't this variable and type be DuplicateName and duplicateName respectively?